### PR TITLE
Change Output Location List

### PR DESCRIPTION
**Pull request by andrew-mestas**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@andrew-mestas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-anmest). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: scripts/unit.js [hunk 1]
```diff
@@ -78,4 +78,4 @@ process.on('exit', function (err) {
   console.log('End:' + new Date().toLocaleTimeString());
 })
 
-require('../node_modules/mocha/bin/mocha');
+require('../node_modules/mocha/bin/_mocha');
```
Comment (amarzavery) [inline scripts/unit.js]: unrequired change

File: lib/commands/arm/sample-service/sample-service.sample._js [hunk 2]
```diff
@@ -0,0 +1,154 @@
+/**
```
Comment (amarzavery) [inline lib/commands/arm/sample-service/sample-service.sample._js]: unrequired file

File: lib/commands/arm/sample-service/sampleUtils._js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (amarzavery) [inline lib/commands/arm/sample-service/sampleUtils._js]: unrequired file

File: lib/commands/arm/providers/provider._js [hunk 4]
```diff
@@ -56,24 +56,61 @@ exports.init = function (cli) {
...
           log.data($('Registration state:'), data.registrationState);
-          log.data('');
-          log.table(data.resourceTypes, function (row, rt) {
-            row.cell($('Resource Type Name'), rt.resourceType);
-            row.cell($('Locations'), rt.locations.join(', '));
-          });
+          log.data($('Resource types:'));
+
+          if (data.resourceTypes && data.resourceTypes.length > 0)
+          {
```
Comment (amarzavery) [inline lib/commands/arm/providers/provider._js]: indentation with 2 spaces only

**Pull request closed without merging**